packages/xchain-ethereum/src/types/client-types.ts

```diff
@@ -1,13 +1,26 @@
 import { ethers } from 'ethers'
 import { BaseAmount } from '@xchainjs/xchain-util'
+import { Balance } from '@xchainjs/xchain-client'
 
 export type Address = string
 
 export enum Network {
-  TEST = 'goerli',
+  TEST = 'kovan',
   MAIN = 'homestead',
 }
 
+export type ETHBalance = Balance & {
+  assetAddress?: string
+  decimals?: number
```

Comment:
`decimals` are part of `amount: BaseAmount` in `Balance` 

That's we can still use `Balance` without additional information.